upgrade/helpers/tasks.py

```diff
@@ -793,8 +793,8 @@ def satellite_upgrade():
                 f'--whitelist="disk-performance" '
                 f'--target-version {os.environ.get("TO_VERSION")}.z -y')
         else:
-            # use beta until 6.8 is GA
-            if os.environ.get('TO_VERSION') == '6.8':
+            # use beta until 6.9 is GA
```

Comment:
TO_VERSION is used several times here and above as well. Initialize the variable for `to_version` instead of fetching it from os env instead.

Not a blocking comment for this PR though.

Reply:
Thanks for notifying me, I will handle it in separate PR.